src/components/Overview/Overview.react.js

```diff
@@ -172,13 +171,13 @@ class Overview extends Component{
 	              <p className="description__text"><b>SUSI.AI</b> already runs on many chat
 	                    services and social networks. We are developing plugins for all
 	                    major services including
-	                    &nbsp;<a href='https://github.com/fossasia/susi_tweetbot'>Twitter</a>,
-                      &nbsp;<a href='https://github.com/fossasia/susi_fbbot'>Facebook</a>,
-	                    &nbsp;<a href='https://github.com/fossasia/susi_linebot'>Line</a>,
-	                    &nbsp;<a href='https://github.com/fossasia/susi_slackbot'>Slack</a>,
-                      &nbsp;<a href='https://github.com/fossasia/susi_wechatbot'>We Chat</a>,
-	                    &nbsp;<a href='https://github.com/fossasia/susi_viberbot'>Viber</a>,
-	                    &nbsp;<a href='https://github.com/fossasia/susi_gitterbot'>Gitter</a>. Just set up SUSI on your channel and add
+	                    &nbsp;<a target="_blank" rel="noopener noreferrer" href='https://github.com/fossasia/susi_tweetbot'>Twitter</a>,
+                      &nbsp;<a target="_blank" rel="noopener noreferrer" href='https://github.com/fossasia/susi_fbbot'>Facebook</a>,
+	                    &nbsp;<a target="_blank" rel="noopener noreferrer" href='https://github.com/fossasia/susi_linebot'>Line</a>,
+	                    &nbsp;<a target="_blank" rel="noopener noreferrer" href='https://github.com/fossasia/susi_slackbot'>Slack</a>,
+                      &nbsp;<a target="_blank" rel="noopener noreferrer" href='https://github.com/fossasia/susi_wechatbot'>We Chat</a>,
+	                    &nbsp;<a target="_blank" rel="noopener noreferrer" href='https://github.com/fossasia/susi_viberbot'>Viber</a>,
+	                    &nbsp;<a target="_blank" rel="noopener noreferrer" href='https://github.com/fossasia/susi_gitterbot'>Gitter</a>. Just set up SUSI on your channel and add
                       &nbsp;<b>@susi</b> in your conversations and SUSI is ready to help.
```

Comment:
Inconsistent indentation.

Reply:
My VC code editor:
<img width="790" alt="screen shot 2018-06-17 at 11 27 47 pm" src="https://user-images.githubusercontent.com/18073126/41510649-174ecdb2-7286-11e8-838a-08909a130e17.png">
I have manually checked and updated it now